expfmt/text_parse_test.go

```diff
@@ -384,6 +385,73 @@ request_duration_microseconds_count 2693
 				},
 			},
 		},
+
+		// 5: Help string present but empty will result in empty help for metric family
```

Comment:
Great to have these test cases. But please end the sentence with a period. (Same below.)